GVFS/GVFS.FunctionalTests/Tests/GitCommands/GitCommandsTests.cs

```diff
@@ -1019,6 +1019,17 @@ public void EditFileNeedingUtf8Encoding()
             GVFSHelpers.ModifiedPathsShouldContain(this.Enlistment, this.FileSystem, relativeGitPath);
         }
 
+        [TestCase]
+        public void ChangeTimestampAndDiff()
+        {
+            // Change the timestamp - The lastwrite time can be close to the time this test method gets
+            // run. Changing (Subtracting) it to the past so there will always be a difference.

```

Comment:
Could you include a few details about scenarios where users would hit this?

In other words, what is the significance of adjusting the last right time?  How does that impact the behavior of `git diff --raw`?

Reply:
Added a brief description.